pkg/proxy/winkernel/hnsV2.go

```diff
@@ -232,16 +232,37 @@ func (hns hnsV2) getLoadBalancer(endpoints []endpointsInfo, flags loadBalancerFl
 		lbFlags |= hcn.LoadBalancerFlagsDSR
 	}
 
-	lb, err := hcn.AddLoadBalancer(
-		hnsEndpoints,
-		lbFlags,
-		lbPortMappingFlags,
-		sourceVip,
-		vips,
-		protocol,
-		internalPort,
-		externalPort,
-	)
+	lbDistributionType := hcn.LoadBalancerDistributionNone 
```

Comment:
Can you just add one more parameter to AddLoadBalancer instead of duplicating this following code?

Reply:
I guess  you have to add one more function in hcsshim  since this is a new field  which  AddLoadBalancer does not have.  You can leave this as is for now.

Reply:
A while ago @ksubrmnn @erfrimod @madhanrm and I discussed this. Since adding a parameter to AddLoadBalancer would be a breaking change, we decided on this solution until we redesign the APIs to be forward compatible.